### ISSUE TITLE
3.4 Grouping Operable Elements

### ISSUE THREAD

**Issue by ciancwright** [ciancwright/accessibility#12]
- Multiple elements performing the same action should be contained in a single actionable element

WCAG 2.0 success criterion:
- 2.4.4 Link Purpose (In Context) (Level A)
- 2.4.9 Link Purpose (Link Only) (Level AA)
